settings.gradle.kts

```diff
@@ -17,10 +17,24 @@ pluginManagement {
     }
 }
 
-include(
-        ":samples:junit-tests",
+include(":samples:junit-tests")
+
+if (isAndroidSDKConfigured()) {
+    include(
         ":samples:kotlin-android",
         ":samples:kotlin-coroutines",
-        ":samples:groovy-android")
+        ":samples:groovy-android"
+    )
+}
 
 includeBuild("gradle-plugin")
+
+fun isAndroidSDKConfigured(): Boolean {
+    if (System.getenv("ANDROID_SDK_ROOT") != null) {
+        return true;
+    }
+    // TODO: Check if `sdk.dir` property is defined in `local.properties`
```

Comment:
You can remove this. If `sdk.dir` property is not defined in `local.properties`, it's up to the user to configure it. The ID or the command line will notify if the environment is misconfigured.

Reply:
The idea of this PR is to conditionally include the projects depending on the presence of the android SDK.
The projects should be included if the Android SDK is available (`ANDROID_SDK_ROOT` env variable or `sdk.dir` property) or it's running on the CI 

Considering this we need read the property to ensure that it is present.

Reply:
I can't find official docs right now, anyway the `local.properties` file will be recreated if missing with content of the `ANDROID_SDK_ROOT`. So it's sufficient to check for the env variables.

Reply:
I've updated the PR reversing the logic (decide, via env variable, if android should be skipped) because there is no way to access to the content of `local.properties` within `settings.gradle.kts`.

This is relevant to me as I might define the Android SDK into the local properties file (and no env variable) and it would be _annoying_ if Android tasks would be ignored.

About _no way to access to the content_, technically there is a hack that can be used (`cat ... | grep sdk.dir`) but it look bad enough to read a file hoping that a string is present.

So I've decided to get a different direction here.